notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>a jupyter widget would be really slick here, letting people adjust the parameters and see the output curves for qps vs # replicas</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>yeah, agreed. that could be really cool. One issue is that we are expecting this to be run in Databricks, so I think that makes it challenging. So, maybe we can set that as a future iteration.</p><p><br></p>

Reply:
makes sense, it actually probably sounds cooler than it would look since this is all linear, showing the tabular for like you have seems fine